content/publishing-yaml/publish-release-notes.md

```diff
@@ -0,0 +1,50 @@
+---
+description: How to pass release notes with successful builds
+title: Publish release notes
```

Comment:
Let's go for the -ing form for consistency. So either Publishing or Passing release notes as the title